tavla/app/(admin)/utils/index.ts

```diff
@@ -242,6 +243,13 @@ export function getFormFeedbackForError(
                 variant: 'error',
             }
         }
+        case 'footer/empty': {
+            return {
+                form_type: 'user',
```

Comment:
Ser vi for det meste kun bruker 'user' når det er snakk om autentisering. Endre til `general`?

Reply:
denne kan fjernes siden vi nå håndterer alle states footeren kan være i og det ikke er feil man kan gjøre her for brukeren